index.html

```diff
@@ -22,7 +23,7 @@
             style="position:absolute;left:0;top:0;"></canvas>
     <canvas id="projectionsLayerMiddle" tabindex="0" width="800" height="300"
```

Comment:
Now make all the background canvas' width 1600 to match your new extended background